### PR TITLE
Update to include Blog Specific Criteria

### PR DESCRIPTION
**Pull request by melba-lopez**
Updates made per statements from community members - https://github.com/slsa-framework/slsa/issues/309#issuecomment-1158087547 ; SLSA 07-07 Community Notes: https://docs.google.com/document/d/1cx3fOBfic6A0xc2on25ITK4vQHUdxgBmJoSS1LPqDJo/edit#heading=h.2eoq8uil87x8

Signed-off-by: Melba <101211710+melba-lopez@users.noreply.github.com>

### CODE REVIEW

File: 6._Contributing.md [hunk 1]
```diff
@@ -83,3 +85,10 @@ The next section contains more information on the workflow followed for Pull Req
 * Pull requests can be closed by the author without merging.
 
 * Pull requests may be closed by a Maintainer if the decision is made that it is not going to be merged.
+
+**4.6 Blog Post Specific**
+
+* A Community/Official Blog Post is considered to be a broad SLSA announcement that may be in regards to a new release, clarification, new tooling, showcase work, etc.  These blog posts are typically in coordination with the broader SLSA community and comes from full community consensus.  PRs must meet all necessary approvals from at least 2-3 steering committee members from differing organizations.  
+
+* Guest Blog Post are considered to be from a single person/organization that is sharing how they are implementing/applying SLSA in their respective project/organization.  PRs must meet all necessary approvals from at least 2-3 steering committee members, which will assess the accuracy of any claims with regards to SLSA compliance.  _Note: The views expressed by Guests are not official positions of the SLSA community or any parent organization. The author has requested and incorporated reviewer feedback whenever possible, but the opinions presented are the author’s alone_
```
Comment (MarkLodato) [inline 6._Contributing.md:93]: Personally I think 1 approval is sufficient for guest posts.

**Review by MarkLodato — APPROVED** [review]
Thanks! Overall looks good, but I do have one question below.

File: 6._Contributing.md (hunk 1 above)
Comment (melba-lopez) [reply 6._Contributing.md:93]: Thanks Mark.  I think that was a quote from David W. to have 2-3 steering committee members.  I can change it to 1.

**Comment by MarkLodato** [comment]
@slsa-framework/slsa-steering-committee any other opinions?

File: 6._Contributing.md [hunk 2]
```diff
@@ -83,3 +85,10 @@ The next section contains more information on the workflow followed for Pull Req
 * Pull requests can be closed by the author without merging.
 
 * Pull requests may be closed by a Maintainer if the decision is made that it is not going to be merged.
+
+**4.6 Blog Post Specific**
+
+* A Community/Official Blog Post is considered to be a broad SLSA announcement that may be in regards to a new release, clarification, new tooling, showcase work, etc.  These blog posts are typically in coordination with the broader SLSA community and comes from full community consensus.  PRs must meet all necessary approvals from at least 2-3 steering committee members from differing organizations.
```
Comment (joshuagl) [inline 6._Contributing.md:91]:
```suggestion
* A Community/Official Blog Post is considered to be a broad SLSA announcement that may be in regards to a new release, clarification, new tooling, showcase work, etc.  These blog posts are typically in coordination with the broader SLSA community and comes from full community consensus.  PRs must meet all necessary approvals from at least 2 steering committee members from differing organizations.  
```
It's better to be specific in how many reviewers we require. For [specification changes](https://github.com/slsa-framework/slsa/blob/main/CONTRIBUTING.md#review-and-approval) we require at least two steering committee members review, I propose we use the same language here.

**Review by joshuagl — COMMENTED** [review]
Thanks for writing this up @melba-lopez! I've made a suggestion that we be specific in how many steering committee reviewers we expect, but otherwise LGTM.

**Comment by joshuagl** [comment]
I'll leave this a day or so and then merge (with my suggestion) if there aren't any objections.

**Comment by melba-lopez** [comment]
Thanks @joshuagl i think this got lost in the sea of PR commits for the blog :).  I pushed your changes.

**Pull request merged**